src/templates/about-page.js

```diff
@@ -2,32 +2,20 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 export const AboutPageTemplate = ({ title, header1, image, text, header2 }) => (
-  <section className="section section--gradient">
-    <div className="container">
-      <div className="columns">
-        <div className="column is-10 is-offset-1">
-          <div className="section">
-            <h2 className="header-title title is-size-3 has-text-weight-bold is-bold-light">
-              {title}
-            </h2>
-            <p className="submeny-placeholder">submeny | placeholder</p>
-            <div className="about-oms">
-              <h2 className="title is-size-3 has-text-weight-bold is-bold-light">
-                {header1}
-              </h2>
-              <div className="content">{text}</div>
-              <img src={image} alt="" />
-            </div>
-            <div className="about-employees">
-              <h2 className="header-title title is-size-3 has-text-weight-bold is-bold-light">
-                {header2}
-              </h2>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </section>
+  <div className="container">
+    <section className="about-submenu">
+      <h2>{title}</h2>
+      <p className="submenu-placeholder">submeny | placeholder</p>
+    </section>
+    <section className="about-oms">
+      <h2>{header1}</h2>
+      <div className="content">{text}</div>
+      <img src={image} alt="" />
```

Comment:
Vi må passe på å få inn alt-tekst for skjermlesere osv.

Reply:
Vi burde nok ha det inn i CMSet ja. 

Men ettersom det ikke er relatert til denne PRen så foreslår vi tar det på et annet issue senere, f.eks. OMS-61